lib/puppet_metadata/github_actions.rb

```diff
@@ -89,17 +105,25 @@ def beaker_os_releases(at = nil)
     end
 
     def puppet_beaker_test_matrix(at)
+      beaker_test_matrix(at, 'puppet')
+    end
+
+    def openvox_beaker_test_matrix(at)
+      beaker_test_matrix(at, 'openvox')
```

Comment:
And following up on the above discussion:
```suggestion
      beaker_test_matrix(at, 'puppet') + beaker_test_matrix(at, 'openvox')
```

You can even keep them alphabetically:
```suggestion
      beaker_test_matrix(at, 'openvox') + beaker_test_matrix(at, 'puppet')
```